plugins/module_utils/mso.py

```diff
@@ -1003,8 +1003,12 @@ def lookup_users(self, users, ignore_not_found_error=False):
 
         ids = []
         if self.platform == "nd":
-            remote_users = self.nd_request("/nexus/infra/api/aaa/v4/remoteusers", method="GET")
-            local_users = self.nd_request("/nexus/infra/api/aaa/v4/localusers", method="GET")
+            remote_users = self.nd_request("/nexus/infra/api/aaa/v4/remoteusers", method="GET", ignore_not_found_error=True)
+            local_users = self.nd_request("/nexus/infra/api/aaa/v4/localusers", method="GET", ignore_not_found_error=True)
+
+            if remote_users == {} and local_users == {}:
+                remote_users = self.nd_request("/api/config/class/remoteusers", method="GET")
+                local_users = self.nd_request("/api/config/class/localusers", method="GET")
```

Comment:
in which version was this api exposed? should we first expose the new api call and then retry the old if already a lot of older versions support it?

same comment applies for lookup_remote_users below

Reply:
do we somehow test that both endpoints are indeed returning the same results for both api's?